main.go

```diff
@@ -106,55 +105,79 @@ func main() {
 		payload.WindowsExporterPath,
 	}
 	if err := checkIfRequiredFilesExist(requiredFiles); err != nil {
-		log.Error(err, "could not start the operator")
-		os.Exit(1)
-	}
-
-	// The watched namespace defined by the WMCO CSV
-	namespace, err := k8sutil.GetWatchNamespace()
-	if err != nil {
-		log.Error(err, "failed to get watch namespace")
+		setupLog.Error(err, "could not start the operator")
 		os.Exit(1)
 	}
 
 	ctx := context.TODO()
 	// Become the leader before proceeding
 	err = leader.Become(ctx, "windows-machine-config-operator-lock")
 	if err != nil {
-		log.Error(err, "failed to become a leader within current namespace")
+		setupLog.Error(err, "failed to become a leader within current namespace")
 		os.Exit(1)
 	}
 
-	// Allow for the watching of cluster-wide resources with "", so that we can watch nodes,
-	// as well as resources within the `openshift-machine-api` and WMCO namespace
-	namespaces := []string{"", "openshift-machine-api", namespace}
-	// Create a new Cmd to provide shared dependencies and start components
-	mgr, err := manager.New(cfg, manager.Options{
-		NewCache:           cache.MultiNamespacedCacheBuilder(namespaces),
+	// Create a new Manager to provide shared dependencies and start components
+	// The NewCache field is not being set, as the default is a cluster wide scope, which is what we want
```

Comment:
This means we will always be hitting the API server. . Is there a reason to not use `cache.MultiNamespacedCacheBuilder(namespaces)` like we were doing before?

Reply:
It seems to me that giving an empty string as one of the arguments to:  `cache.MultiNamespacedCacheBuilder()` is going to result in an informer being created that watches all namespaces.

[In the constructor, MultiNamespacedCacheBuilder](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/cache/multi_namespace_cache.go#L49-L51):
```
		for _, ns := range namespaces {
			opts.Namespace = ns
			c, err := New(config, opts)
```
For each namespace given a new cache is being created.

The `Namespace` field of cache.Options has [this documentation](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/cache/cache.go#L103-L105)
```
	// Namespace restricts the cache's ListWatch to the desired namespace
	// Default watches all namespaces
	Namespace string
```
Following the [New Cache function](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/cache/cache.go#L116) logic, down to the creation of the [InformersMap](https://github.com/kubernetes-sigs/controller-runtime/blob/11dfabf5e2e83cdb8cba28fedcda0ae5e10474f0/pkg/cache/internal/deleg_map.go#L54-L60), into the invocation of the [newSpecificInformersMap](https://github.com/kubernetes-sigs/controller-runtime/blob/11dfabf5e2e83cdb8cba28fedcda0ae5e10474f0/pkg/cache/internal/deleg_map.go#L108-L120), this is expanded on by the documentation of the [namespace field](https://github.com/kubernetes-sigs/controller-runtime/blob/11dfabf5e2e83cdb8cba28fedcda0ae5e10474f0/pkg/cache/internal/informers_map.go#L120-L122)
```
	// namespace is the namespace that all ListWatches are restricted to
	// default or empty string means all namespaces
	namespace string
```

So with that being said, it makes more sense to have the default `all namespace` cache/informer, instead of using the `MultiNamespacedCache` which results in an `all namespace` cache/informer, as well as redundant caches/informers for the other namespaces we would provided `wmco/mapi`

Reply:
Thanks for digging deeper into. 

>So with that being said, it makes more sense to have the default all namespace cache/informer, instead of using the MultiNamespacedCache

What does this entail? Not using the `NewCache` field as you are doing now?

Reply:
Yes exactly, without providing a`NewCache` value, the manager will create its own cache which will watch all namespaces